go.mod

```diff
@@ -67,24 +67,24 @@ require (
 	go.uber.org/zap v1.23.0 // indirect
 	golang.org/x/crypto v0.0.0-20220829220503-c86fa9a7ed90 // indirect
 	golang.org/x/net v0.3.1-0.20221206200815-1e63c2f08a10 // indirect
-	golang.org/x/oauth2 v0.0.0-20220909003341-f21342109be1 // indirect
+	golang.org/x/oauth2 v0.2.0 // indirect
```

Comment:
Actually I think most of these go.mod changes are unrelated and this one in particular is causing an issue in operator-lint. Can you please rework this so that only lib-common is refreshed to the current commit?

Reply:
OK, I think the mod.* changes can be removed from this PR entirely now